book/src/csi-driver-object.md

```diff
@@ -45,6 +48,12 @@ There are three important fields:
     * `"csi.storage.k8s.io/pod.namespace": pod.Namespace`
     * `"csi.storage.k8s.io/pod.uid": string(pod.UID)`
   * For more information see [Pod Info on Mount](pod-info.md).
+* `volumeLifecycleModes`
+  * This field was added in Kubernetes 1.16 and must not be set when using an older Kubernetes release.
```

Comment:
must => cannot

Reply:
I was thinking "must not be set in the .yaml file because it's invalid", but "cannot" is simpler. Changed.